.stylelintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "stylelint-config-standard"
+}
```

Comment:
Seems your editor is configured wrongly. Please ensure there's a new line at the end of each file.